.gitlab-ci.yml

```diff
@@ -201,6 +201,23 @@ check-wasm:
         cargo check --verbose --no-default-features --target wasm32-unknown-unknown --manifest-path ./crates/${crate}/Cargo.toml;
       done
 
+dylint:
+    stage:                           check
+    <<:                              *docker-env
+    <<:                              *test-refs
+    script:
+    - cd linting/
+    # we are using a toolchain file in this directory
+    # add required components for CI
+    - rustup component add rustfmt clippy
+    - cargo check --verbose
+    - cargo fmt --verbose --all -- --check
+    - cargo clippy --verbose -- -D warnings;
+
```

Comment:
```suggestion
```
We tend not to divide the `script` section with empty lines in `.gitlab-ci.yml`. I can't say that it's a strict rule but would be nice to have the same format.